peekaboo/toolbox/cortex.py

```diff
@@ -152,10 +268,32 @@ class HybridAnalysis(CortexFileAnalyzer):
 
 class VirusTotalQueryReport(CortexAnalyzerReport):
     """ Represents a Cortex VirusTotal_GetReport_3_0 analysis JSON report. """
-    def __init__(self, report):
-        self.report = report
+    report_schema = schema.Schema({
+        "summary": {
+            "taxonomies": [
+                {
+                    "level": schema.Or("info", "malicious", "safe"),
+                    "namespace": schema.Schema("VT"),
```

Comment:
Does string "VT" need to be a `schema.Schema` object?

Reply:
No

Reply:
No